website/source/docs/auth/userpass.html.md

```diff
@@ -92,3 +92,214 @@ $ vault write auth/userpass/users/mitchellh \
 The above creates a new user "mitchellh" with the password "foo" that
 will be associated with the "root" policy. This is the only configuration
 necessary.
+
+## API
+
+### /auth/userpass/users/[username]
+#### POST
+
+<dl class="api">
+  <dt>Description</dt>
+  <dd>
+      Create a new user or update an existing user.
```

Comment:
You should make it clear here that the distinction between create and update is honored. Stolen from elsewhere:

This path honors the distinction between the `create` and `update` capabilities inside ACL policies.